src/games/hostgamewidget.py

```diff
@@ -1,5 +1,5 @@
 from PyQt5 import QtCore
-import modvault
+from modvault import utils
```

Comment:
Same here, I'd rather keep the 'modvault' prefix to avoid confusing 'utils' with a local variable.